Lib/asyncio/base_events.py

```diff
@@ -561,8 +561,13 @@ async def shutdown_asyncgens(self):
                     'asyncgen': agen
                 })
 
-    async def shutdown_default_executor(self):
-        """Schedule the shutdown of the default executor."""
+    async def shutdown_default_executor(self, timeout=None):
+        """Schedule the shutdown of the default executor.
+
+        The timeout parameter specifies the amount of time the threadpool will
```

Comment:
threadpool -> executor (also later in this docstring and again in the warning message)

Reply:
done